client/src/components/select/index.js

```diff
@@ -1,11 +1,19 @@
-import React, { Component } from "react";
+import React from "react";
+import PropTypes from "prop-types";
 
 import "./select.css";
 
-const Select = ({ labelText, firstItem, cities, name, onSelectChange }) => (
+const Select = ({
+  labelText,
+  className,
+  firstItem,
+  cities,
+  name,
+  onSelectChange
+}) => (
   <div className="container-select">
     <label className="label-style">{labelText}</label>
-    <select className="select-style" onChange={onSelectChange} name={name}>
+    <select className={className} onChange={onSelectChange} name={name}>
```

Comment:
You can have 'select-style' and className combined here instead of always passing 'select-style' as part of the className whenever you use this Select component.

Here's a helpful little library for combining class names: https://www.npmjs.com/package/classnames
You can pass multiple class names as arguments and it will combine them for you (and remove falsy ones automatically):
`<select className={classnames('select-style', className)} ...`

And you can even pass an object of class names mapped to booleans, indicating if they should be added or not, which is really useful:
```
<SomeComponent 
  className={classnames('some-cool-component', {
    'is-loading': this.props.isLoading, 
    'is-dark': this.props.theme === 'dark'
  })}
  ...
```

Reply:
I think you could do the same for 'input-style' being passed to all of the `<input` components too.

Reply:
Really this note is very good :+1: 
we have done it
Thanks @tomduggan85

Reply:
I discover that it will be change the style because the customer style is different 
so it will not be useful if I put it but i will search if it needed in other components :+1:  @tomduggan85

Reply:
Ah ok  ¯\_(ツ)_/¯
Thanks for checking!

Reply:
Oh good!  I'm glad the reviews are helping.
And no need to apologize, you can think of all of my comments as suggestions or ideas - don't feel like you have to make any changes.  If it helps to read them and gain some knowledge, even though you don't have time to adjust the code, then that's great!  
It's a good skill to know when the code is in a good enough state and it's time to move on to the next task, instead of perfecting the code but never shipping a product :) It's something we have to deal with every day at work.